client.go

```diff
@@ -0,0 +1,179 @@
+package luasandbox
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/calyptia/api/types"
+)
+
+type Client struct {
+	url        string
+	httpClient http.Client
+	nextId     int
+}
+
+type params struct {
+	Events []types.FluentBitLogAttrs `json:"events"`
+	Filter string                    `json:"filter"`
```

Comment:
```suggestion
	Script string                    `json:"script"`
```

I would rename this to `script` or even `code`. But "filter" doesn't make too much sense.

Reply:
I really don't mind changing the name, the problem is that this will also have to be changed in the API server and all its tests. Can we handle these aesthetic separately?